app/controllers/discover.js

```diff
@@ -49,6 +49,7 @@ export default ApplicationController.extend({
     numberOfResults: 0,
     took: 0,
     numberOfSources: 0,
+    types: '',
```

Comment:
Nope. Long story short, I changed it from an empty object to a string when debugging something and never changed it back.